packages/bpk-component-chip-group/src/BpkChipGroup.tsx

```diff
@@ -218,16 +218,16 @@ const WrapChipGroup = ({
     chips={chips}
     label={label} />;
 
-const BpkChipGroup = ({ label, type, ...rest }: ChipGroupProps) => {
+const BpkChipGroup = (props: ChipGroupProps) => {
+  const { type } = props;
```

Comment:
We want to narrow the props object below using conditional logic so keeping whole object intact